### PR TITLE
Add d3kit@3.2.0 w/ git auto-update config

### LINKED ISSUE #9928: [Request] Add d3kit
**Library name:** d3kit
**Git repository url:** https://raw.githubusercontent.com/twitter/d3kit/master/dist/d3kit.min.js
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9928

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/twitter/d3kit/
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: [MIT](https://github.com/twitter/d3kit/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 122
   - Count of stars: 283
   - Count of forks: 35
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9928, cc @twitter

### CODE REVIEW

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

File: ajax/libs/d3kit/3.2.0/d3kit-es.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// Error: Unexpected token: punc ({)
```
Comment (PeterDaveHello) [inline ajax/libs/d3kit/3.2.0/d3kit-es.min.js]: Looks like the minify process failed, let's remove it.

Comment (extend1994) [inline ajax/libs/d3kit/3.2.0/d3kit-es.min.js]: @PeterDaveHello Do you mean remove `d3kit-es.min.js`, which would only left d3kit-es.js?

Comment (PeterDaveHello) [inline ajax/libs/d3kit/3.2.0/d3kit-es.min.js]: Yep.

**Comment by extend1994** [comment]
@PeterDaveHello I removed `d3kit-es.min.js`, please help review this, thank you.

**Comment by maruilian11** [comment]
please fill the npm info if it exist.
https://www.npmjs.com/package/d3kit

**Comment by maruilian11** [comment]
LGTM

**Comment by maruilian11** [comment]
@cdnjs/intern2 please help me make the double reivew, thanks!

**Comment by PeterDaveHello** [comment]
@extend1994 you should tag the author, not organization, it'll not work at all.

**Comment by extend1994** [comment]
@maruilian11 I filled it, thanks for reminder.
@PeterDaveHello Okay, I will do it afterward.

**Pull request merged**